### DISCUSSION TITLE
No Email sent for Password Reset

### DISCUSSION THREAD

**Discussion by DeepinderSingh-057** [community/community#134962] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### GitHub Feature Area

Actions

### Body

I been trying to login to my github account . But i forgot my password when i clicked on forgot password and entered my email  and  clicked on send email and password reset . The page jump to home page . 

I checked my mail box too (primary,bin,spam) but didn't find email 

My last  email which GitHub sent me was 23/12/2023

My username is @DeepinderSingh-555
My email id is DEEPINDER.58-21CSE@mbscet.edu.in 

Help me guys

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @DeepinderSingh-057 ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Comment by Ali-Iqbal-0** [comment]
hello sir You should contact with Github support & tell them your issue related to it they will help you to solve it

**Comment by ebndev** [comment]
Hi @DeepinderSingh-057,

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!

**Reply by Kikofury** [reply]
@talum @talum 
![IMG_20220310_234615~3.jpg](yes)

- [ ]
